roles/openshift_openstack/vars/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+stack_name: "{{ env_id }}.{{ public_dns_domain }}"
```

Comment:
Some of these variables are not namespaced well.  Also, do we need to put these in vars/main.yml?  Could they live in defaults/main.yml?  That will allow them to be overriden by inventory.

Probably some of these variables don't need to be defined, we can just use the "openstack_x" variables in their places.

Reply:
+1 good point with using defaults instead of vars. Ansible has a tricky precedence, so let's follow this recommendation

Reply:
This is my last 'big item' for cleanup.